### PR TITLE
89 Define and implement Payment endpoints

### LINKED ISSUE #89: Define and implement payment endpoints
ℹ️ This issue is related to the Epic #75

### Describe the solution you'd like

Define and implement endpoints for payments.

### Describe alternatives you've considered

* The necessary resources (Controllers, Plugs, Views) must be defined and implemented.
* Create the test suite for the endpoints

### PR DESCRIPTION
**Pull request by CristhianRodriguezMolina**
## Description

Fixes #89

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by einerzg — APPROVED** [review]
🦾

**Review by miguelnietoa — COMMENTED** [review]
Nice job, guys! 👌🏻 
Just a comment. ⬇️

File: apps/mintacoin_web/lib/mintacoin_web/views/payments_view.ex [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        }
+      }) do
+    %{
+      resource: "payment",
+      source: source,
+      destination: destination,
+      amount: amount,
+      asset_code: asset_code
+    }
+  end
```
Comment (miguelnietoa) [inline apps/mintacoin_web/lib/mintacoin_web/views/payments_view.ex:22]: I think the `id` should be returned, as it is necessary for the `GET /payments/:id`

File: apps/mintacoin_web/test/mintacoin_web/controllers/payments_controller_test.exs [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+  describe "retrieve payment" do
+    test "returns payment with valid given id", %{conn: conn} do
+      %{
+        id: payment_id,
+        source: source,
+        destination: destination,
+        asset_code: asset_code,
+        amount: amount
+      } = insert(:payment)
```
Comment (miguelnietoa) [inline apps/mintacoin_web/test/mintacoin_web/controllers/payments_controller_test.exs:143]:
```suggestion
      %Payment{
        id: payment_id,
        source: source,
        destination: destination,
        asset_code: asset_code,
        amount: amount
      } = insert(:payment)
```

**Review by miguelnietoa — CHANGES_REQUESTED** [review]

**Review by miguelnietoa — APPROVED** [review]
Awesome! 🚀 Thanks

**Pull request merged**